### ISSUE TITLE
Compilation error in 0.239

### ISSUE THREAD

**Issue by 0-wiz-0** [mamedev/mame#9057]
0.238 compiled fine, but compiling 0.239 on NetBSD fails with:
```
Compiling src/devices/bus/a2bus/nippelclock.cpp...                                                                                                                               
In file included from ../../../../../src/emu/softlist.h:18,
                 from ../../../../../src/emu/softlist_dev.h:17,                                                                                                                  
                 from ../../../../../src/devices/imagedev/harddriv.h:15,                                                                                                         
                 from ../../../../../src/devices/machine/corvushd.h:20,
                 from ../../../../../src/devices/bus/a2bus/a2corvus.h:18,                                                                                                        
                 from ../../../../../src/devices/bus/a2bus/cards.cpp:19:
../../../../../src/emu/romentry.h:12:2: error: #error Dont include this file directly; include emu.h instead.                  
   12 | #error Dont include this file directly; include emu.h instead.                                                                                                           
      |  ^~~~~                                                                          
In file included from ../../../../../src/devices/bus/a2bus/cards.h:15,
                 from ../../../../../src/devices/bus/a2bus/cards.cpp:12:                                                                                                         
../../../../../src/devices/bus/a2bus/a2bus.h:29:18: error: 'uint32_t' does not name a type                                                                                       
   29 | static constexpr uint32_t A2BUS_7M_CLOCK = 7159090;                             
      |                  ^~~~~~~~                                                                                                                                                
../../../../../src/devices/bus/a2bus/a2bus.h:38:42: error: expected class-name before ',' token                                                                                  
   38 | class a2bus_slot_device : public device_t, public device_single_card_slot_interface<device_a2bus_card_interface>                                                         
      |                                          ^                   
../../../../../src/devices/bus/a2bus/a2bus.h:38:84: error: expected template-name before '<' token                             
   38 | class a2bus_slot_device : public device_t, public device_single_card_slot_interface<device_a2bus_card_interface>                                                         
      |                                                                                    ^                                                                                     
...
```
and many more errors.

**Comment by bigmistercactus** [comment]
I don't think this OS is supported

**Comment by rb6502** [comment]
BSDs are not directly supported, but there's no reason it shouldn't work either.

**Comment by bigmistercactus** [comment]
So as I understzand correctly, any GCC compiler should work?

**Comment by cuavas** [comment]
FreeBSD and NetBSD are supported, in the sense that we accept reports for issues.  They just aren’t actively tested as part of our build matrix.

Someone forgot to add the PCH to bus/a2bus/cards.cpp and someone else didn’t notice when cleaning up that file.  This causes build failures if you aren’t using implicit prefix headers.  Should be fixed by 0288a54ca21cb4e4d63e98507060af05abe59768.

**Comment by 0-wiz-0** [comment]
Thanks, @cuavas , the patch does indeed fix the problem for me.
